site/app/views/forum/ForumThreadView.php

```diff
@@ -624,14 +624,19 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
                         }
 						if($thread['status'] !=0) {
 							if($thread['status'] == 1) {
-								$fa_icon = "fa-check-circle";
-								$fa_color = "palegreen";
+								$fa_icon = "fa-check";
+								$fa_color = "5cb85c";
```

Comment:
Please include a # before the color hex.

Reply:
Sorry, my bad. I have '#' before color hex value now.